promotion/order.go

```diff
@@ -10,6 +13,26 @@ import (
 // Delete this file once the issue is completed
 // https://github.com/brave-intl/bat-go/issues/263
 
+// NullString is a type that lets ya get a null field from the database
+type NullString struct {
```

Comment:
might be cool to put this in `utils/datastore/models.go` or something... We are defining this twice in bat-go.

Reply:
https://github.com/brave-intl/bat-go/pull/308/commits/f7acb87cd3de177488f862891393e0e50f6581f3